greet.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+module.exports = exports = {};
```

Comment:
This functionality, should be within your greet function, if you want to get the bonus.